packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -766,6 +839,39 @@ export interface RollingUpdateConfiguration {
   readonly suspendProcesses?: ScalingProcess[];
 }
 
+/**
+ * A group of notificationTypes, you can use one of the predefined groups, such as NotificationTypes.ERRORS
+ * or create a custome group by instantiating a `NotificationTypes` object, e.g: `new NotificationTypes(`NotificationType.INSTANCE_LAUNCH`)`.
+ */
+export class NotificationTypes {
```

Comment:
consider calling this NotificationType (singular) which contains `FleetScalingEvents`.
I know the Cfn properties are called `notificationTypes` and another called `notificationConfigurations`, but I think we can be simpler

Reply:
changed to ScalingEvent and ScalingEvents